misc/tests/dir/hello/3.txt

```diff
@@ -27,6 +27,7 @@ abcdabcd
 abcdabcd
 abcdabcd
 abcdabcd
+aaaa
```

Comment:
What is this file? and why is it full of random garbage

Reply:
It's a test file for `checks_test.go`.